apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -3148,10 +3145,6 @@ naming_system_broken_txs(_Config) ->
     {ok, []} = rpc(aec_tx_pool, peek, [infinity]),
 
     %% Try to submit txs with empty account
-    {ok, 400, #{<<"reason">> := <<"Name validation failed with a reason: registrar_unknown">>}} =
```

Comment:
Was removed because of same reasons as above - we don't have static list of valid registrars anymore, and can't do this validation at TX creation time.